pkg/client/typed/dynamic/client.go

```diff
@@ -112,12 +126,16 @@ func (rc *ResourceClient) List(opts v1.ListOptions) (*runtime.UnstructuredList,
 	return result, err
 }
 
+func (rc *ResourceClient) get(name string) *restclient.Request {
```

Comment:
I'm not sure this is worth it. It's only used in 2 places, and the function name is inscrutable.